shell/middleware/authenticated.js

```diff
@@ -17,52 +15,7 @@ export default async function({
   route, store, redirect, from, $plugin, next
 }) {
   if ( store.getters['auth/enabled'] !== false && !store.getters['auth/loggedIn'] ) {
```

Comment:
This is still necessary because we don't always proceed any further in the middleware depending on whether the user was authenticated or not.